### PR TITLE
[Issue_602] Add WildFly 33 to the downloads page along with the relea…

### LINKED ISSUE #602: WildFly 33 release announcement
I'm using this issue to record things to mention in the WF 33 release announcement.

Incompatibilities:

https://issues.redhat.com/browse/WFLY-19287

### PR DESCRIPTION
**Pull request by bstansberry**
…se announcement

Resolves #602

### CODE REVIEW

File: _posts/2024-07-23-WildFly33-Released.adoc [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+=== Jakarta EE
+
+Standard WildFly 33 is a compatible implementation of the EE 10 link:https://jakarta.ee/specifications/platform/10/[Platform] as well as the link:https://jakarta.ee/specifications/webprofile/10/[Web Profile] and the link:https://jakarta.ee/specifications/coreprofile/10/[Core Profile]. WildFly is EE 10 Platform, Web Profile and Core Profile compatible when running on Java SE 11, Java SE 17 and Java SE 21.
+
+Evidence supporting our certification is available in the link:https://github.com/wildfly/certifications/tree/EE10[WildFly Certifications] repository on GitHub:
+[cols=",",options="header"]
+|=======================================================================
+|Specification |Compatibility Evidence
+.2+.<| Jakarta EE 10 Full Platform
```
Comment (jamezp) [inline _posts/2024-07-23-WildFly33-Released.adoc]: This should be `.3+`

File: _posts/2024-07-23-WildFly33-Released.adoc [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+| link:https://github.com/wildfly/certifications/blob/EE10/WildFly_33.0.0.Final/jakarta-full-platform-jdk11.adoc#tck-results[SE 11]
+| link:https://github.com/wildfly/certifications/blob/EE10/WildFly_33.0.0.Final/jakarta-full-platform-jdk17.adoc#tck-results[SE 17]
+| link:https://github.com/wildfly/certifications/blob/EE10/WildFly_33.0.0.Final/jakarta-full-platform-jdk21.adoc#tck-results[SE 21]
+.2+.<|  Jakarta EE 10 Web Profile
```
Comment (jamezp) [inline _posts/2024-07-23-WildFly33-Released.adoc]: Same here, `.3+`.

File: _posts/2024-07-23-WildFly33-Released.adoc [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+|Jakarta Messaging| 3.1 |10 & 11
+
+| Jakarta MVC
+(_preview stability only_)| 2.1| N/A xref:note2[^2^]
+
+|Jakarta Pages| 3.1 |10
+
+|Jakarta Persistence| 3.2.0 |11
+
+|Jakarta RESTful Web Services| 3.1 |10
```
Comment (jamezp) [inline _posts/2024-07-23-WildFly33-Released.adoc:160]: This should be 4.0 and 11 for the EE version.

File: _posts/2024-07-23-WildFly33-Released.adoc [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+<dependency>
+    <groupId>org.jboss.arquillian.protocol</groupId>
+    <artifactId>arquillian-protocol-servlet-jakarta</artifactId>
+    <version>1.9.0</version>
+</dependency>
+<dependency>
+    <groupId>org.jboss.arquillian.junit5</groupId>
+    <artifactId>arquillian-junit5-container</artifactId>
+    <version>1.9.0</version>
+</dependency>
```
Comment (jamezp) [inline _posts/2024-07-23-WildFly33-Released.adoc:234]: Minor nit, these should all probably have `<scope>test</scope>`.

**Review by jamezp — CHANGES_REQUESTED** [review]

File: _posts/2024-07-23-WildFly33-Released.adoc (hunk 3 above)
Comment (bstansberry) [reply _posts/2024-07-23-WildFly33-Released.adoc]: Thanks; fixed.

File: _posts/2024-07-23-WildFly33-Released.adoc (hunk 4 above)
Comment (bstansberry) [reply _posts/2024-07-23-WildFly33-Released.adoc]: Definitely. Fixed.

**Review by jamezp — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @jamezp

**Pull request merged**